CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # bedrock-vc-issuer ChangeLog
 
+## 21.1.0 - 2023-TBD
+
+### Added
+- Add support for `@digitalbazaar/ecdsa-2019-cryptosuite` suite when issuing.
+
+### Changed
+- Update peerdep `@bedrock/did-io` to v10.0.
```

Comment:
This is a breaking change.

```suggestion
- **BREAKING** - Update peerdep `@bedrock/did-io` to v10.0.
```

Reply:
Updated [0aa1e32](https://github.com/digitalbazaar/bedrock-vc-issuer/pull/113/commits/0aa1e32f1e82ddbe200e3c9b0cff763aeed3c599)